tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -87,7 +90,7 @@ def _parse_links(self, header: str) -> dict[str | None, dict[str, str | None]]:
             attrs["href"] = url
         return links
 
-    def get_response(self, *args, **kwargs):
+    def get_response(self, *args: Organization, **kwargs: Any) -> Response:
```

Comment:
typically these are `HttpResponseBase` but I guess the type checker let this happen :D